app/background/wallet/service.js

```diff
@@ -594,8 +596,15 @@ class WalletService {
       type: SET_FEE_INFO,
       payload: { fees },
     });
+  };
 
-    await this.node.wdb.addBlock(entry, txs);
+  onNewBlock = async (entry, txs) => {
+    await this._ensureClient();
+    await this.refreshNodeInfo();
+
+    if (entry && txs) {
+      await this.node.wdb.addBlock(entry, txs);
```

Comment:
Doesn't the wallet node do this automatically?

Reply:
Yep - i am stubbing the handler to call `refreshNodeInfo` on new block vs on an interval.